sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+This Cartesian dualism that Chun refers to, source/execution, haunts computation to its core. It is implicated in a related binary that Chun overlooks, the separation of _code_ from _data_. These dualisms crucially influence the way in which software's form and potential is commonly understood today, and account for how its materiality is often overlooked or fails to actualize at all. Conventional source code, before it can be used, must be transformed into something different: a binary executable created by a _compiler_. This transformation is unidirectional; as Chun suggests, it is an act of erasure that enacts the practical gap between source code, which is understood as fundamentally changeable and authorable--the "source" of creative power--and execution, which freezes the creative act and transforms it into an object and product. While code is apparently iterative and endlessly modifiable (by those with the skills and access, anyway), executables are not. One cannot unravel a piece of software and turn it into something else.
+
+Software has no equivalent to Amy Twigger Holroyd's _reknitting_, where existing knitted garments (most strikingly, even those that were mass-produced by machines) are modified or transformed in ways that were unanticipated by their original designers. Re-knitting, according to Twigger Holroyd, involves "a broad range of processes which can be carried out by individuals to alter existing items of knitwear. Re-knitting shifts a knitter's practice from the creation of new items to the re-making of existing pieces" ("About Re-Knitting"). These processes include unravelling, cutting, grafting, insertions, stitch hacks, and replacements (Twigger Holroyd, "The Spectrum"). With these techniques, Twigger Holroyd and her friends make marvelously uncanny and expressive machine/hand hybrid garments by grafting new designs into old jumpers and socks.
+
+![Re-knitted cardigan by Amy Twigger Holroyd](../images/twigger-holroyd-sampler-last-step.jpg)
+_Figure x: A reknitted cardigan by Amy Twigger Holroyd, which was created using a variety of slicing, unravelling, stitch-pickup and grafting processes in order to modify an existing, machine-made garment with her own hand-made alterations. Image courtesy of Amy Twigger Holroyd._
+
+As an art form, re-knitting shows how new works can be created from old ones not just by reusing patterns or schematics, but at the material level. Using Wark's terminology, a reknitted garment represents not only the _content_ of the vector of knitting, but itself a new vector, or _form_, for the creation of another work from and within it. This newness is not just the result of a process of copying, quotation, or appropriation, but of the virtuality of the artifact itself--its ability to engender new forms and futures, "the immaterial virtuality of the material" (Hacker Manifesto 100).
```

Comment:
Comment from David: "First Wark quote: it’s not clear what the ‘term’ is. assuming it is vectors, you should quickly explain its significance as soon as you introduce it. (or maybe you’ve introduced it earlier?)."